packages/cli/src/lib/pg/fetcher.ts

```diff
@@ -10,6 +13,21 @@ export type TransferSchedule = {
   timezone: string,
 }
 
+type AddOnAttachment = {
+  addon: {
+    id: string;
+    name: string;
+    plan: {
+      name: string;
+    };
+  };
+  app: {
+    id: string;
+    name: string;
+  };
+  config_vars: string[];
+}
+
```

Comment:
Instead of duplicating the `AddonAttachment` type, consider creating a new one that is an intersection of types bound to the schema. e.g.
```
type AddOnWithPlan = Heroku.AddOnAttachment['addon'] & {plan: Heroku.AddOn['plan']}
type ExpandedAddOnAttachment = Heroku.AddOnAttachment & {addon: AddonWithPlan, config_vars: Heroku.AddOn['config_vars']}
```

Reply:
Great suggestion, thanks!